var/spack/repos/builtin/packages/rocclr/package.py

```diff
@@ -13,37 +13,67 @@ class Rocclr(CMakePackage):
        runtimes to work on Windows as well as on Linux without much effort."""
 
     homepage = "https://github.com/ROCm-Developer-Tools/ROCclr"
-    url      = "https://github.com/ROCm-Developer-Tools/ROCclr/archive/roc-3.5.0.tar.gz"
 
     maintainers = ['srekolam', 'arjun-raj-kuppala']
 
+    def url_for_version(self, version):
+        # Fix up a typo in the 3.5.0 release.
+        if version == Version('3.5.0'):
+            return "https://github.com/ROCm-Developer-Tools/ROCclr/archive/roc-3.5.0.tar.gz"
+
+        url = "https://github.com/ROCm-Developer-Tools/ROCclr/archive/rocm-{0}.tar.gz"
+        return url.format(version)
```

Comment:
I wonder if they could just fix the tarball name for us.

Reply:
Let me open an issue